spec/index.bs

```diff
@@ -365,32 +378,41 @@ of CPU execution of Javascript.
 
 ## Privacy ## {#security-privacy}
 
+Table-stakes for WebGPU are matching the privacy guarantees of WebGL, but we are looking
+for opportunities where we can futher raise the bar on privacy.
+While these new APIs are powerful and valuable, our users' privacy is essential.
+We're working with our fellow standards body participants to ensure that
```

Comment:
"We" in this context *is* the standards body participants. Maybe reword e.g.
```suggestion
The standards body works to ensure that
```

Reply:
I don't know that this section should talk about us working with ... Maybe "WebGPU is designed so it can be implemented in a manner that makes it not a vector for tracking users"?